### PR TITLE
Add tutorial for using Azure Blob Storage with TensorFlow

### PR DESCRIPTION
**Pull request by yongtang**
This PR adds tutorial for using Azure Blob Storage (emulator) with TensorFlow

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by yongtang** [comment]
@MarkDaoust @lamberta Please take a look.

File: docs/tutorials/azure.ipynb [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup and Usage"
```
Comment (lamberta) [inline docs/tutorials/azure.ipynb]:
```suggestion
        "## Setup and usage"
```

File: docs/tutorials/azure.ipynb [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install required Packages, and restart runtime"
```
Comment (lamberta) [inline docs/tutorials/azure.ipynb]:
```suggestion
        "### Install required packages, and restart runtime"
```

File: docs/tutorials/azure.ipynb [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install and Setup Azurite (optional)\n",
```
Comment (lamberta) [inline docs/tutorials/azure.ipynb]:
```suggestion
        "### Install and setup Azurite (optional)\n",
```

Google dev docs style guide is sentence-case for all titles and headings, see: https://developers.google.com/style/capitalization

File: docs/tutorials/azure.ipynb [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+        "id": "h21RdP7meGzP",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import os\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_io as tfio\n",
+        "\n",
+        "# Swith to False to use Azure Storage instead:\n",
```
Comment (lamberta) [inline docs/tutorials/azure.ipynb]:
```suggestion
        "# Switch to False to use Azure Storage instead:\n",
```

File: docs/tutorials/azure.ipynb [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "acEST3amdyDI",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Read and Write Files to Azure Storage with TensorFlow\n",
```
Comment (lamberta) [inline docs/tutorials/azure.ipynb]:
```suggestion
        "### Read and write files to Azure Storage with TensorFlow\n",
```

**Comment by yongtang** [comment]
@lamberta Thanks! The PR has been updated with all comments addressed. Please take a look and let me know if there are any other issues.

**Pull request merged**